config/routes.rb

```diff
@@ -10,7 +10,7 @@
   resource :registration, only: [:new, :create]
   resource :session, only: [:new, :create, :destroy]
   resource :password, only: [:edit, :update]
-  resource :profile, only: [:edit, :update, :show]
+  resource :profile, only: [:show, :edit, :update], param: :uuid
```

Comment:
should this be `resources` instead of `resource` ?

Reply:
opted for having resource and resources, like that the profile show has :uuid in the url